Flabbergasted-Flamingos/bangazon-commerce-app
# update templates/configure list index

all html templates have been modified

## Problem to Solve
product list should be visible on index view
## Proposed Changes

product url is missing and it has replaced the index url


## Steps to Test Solution

git fetch --all
git checkout navbar
test all urls
navbar should be in tact on every template EXCEPT register/login
